cms/djangoapps/contentstore/views/item.py

```diff
@@ -675,6 +675,21 @@ def _get_source_index(source_usage_key, source_parent):
         return None
 
 
+def is_source_item_in_target_parents(source_item, target_parent):
+    """
+    Returns True if source item is found in target parents otherwise False.
+
+    Arguments:
+        source_item (XBlock): Source xblock.
+        target_parent (XBlock): Target XBlock.
+    """
+    target_ancestors = _create_xblock_ancestor_info(target_parent, is_concise=True)['ancestors']
```

Comment:
Confirm this and see how we can avoid such usages which can break the code in some unseen scenario.